TESTING.md

```diff
@@ -2,6 +2,33 @@
 
 This document provides detailed instructions and procedures for manually testing the various functionalities of Nachet, ensuring that all features operate correctly and as expected before deployment or release.
 
+## Test Case: Backend URL Warning Functionality
+
+**Objective:** Verify the application correctly displays a warning if the backend URL is not set or is non-responsive.
+
+**Preconditions:**
+- [ ] The application is open.
+- [ ] Nachet Backend is set up and running locally. The Nachet Backend can be found at [Nachet Backend GitHub Repository](https://github.com/ai-cfia/nachet-backend). To run the application locally, use the command `hypercorn -b :8080 app:app` in the terminal.
```

Comment:
>80

Reply:
what does this mean @rngadam?

Reply:
means the document is not wrapping at column 80